src/main/resources/swagger/api-docs.yaml

```diff
@@ -4080,6 +4080,81 @@ paths:
           - email
           - profile
 
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/snapshots:
+    post:
+      x-passthrough: true
+      x-passthrough-target: rawls
+      responses:
+        201:
+          description: Created
+          schema:
+            $ref: '#/definitions/WMDataReferenceResponse'
+        404:
+          description: Workspace or snapshot not found
+      parameters:
+        - in: path
+          description: workspace namespace
+          name: workspaceNamespace
+          required: true
+          type: string
+        - in: path
+          description: workspace name
+          name: workspaceName
+          required: true
+          type: string
+        - in: body
+          description: Reference to the Data Repo snapshot
+          name: dataRepoSnapshot
+          required: true
+          schema:
+            $ref: '#/definitions/DataRepoSnapshot'
+      tags:
+        - Snapshots
+      summary: Add a reference to a snapshot in the Terra Data Repo
+      operationId: createDataRepoSnapshot
+      security:
+        - googleoauth:
+            - openid
+            - email
+            - profile
```

Comment:
since this is the default (see line 18), you can omit the `security:` section entirely